### PR TITLE
feat(zipkin) override unknown-service-name

### PR DESCRIPTION
**Pull request by jeremyjpj0916**
Optional configurable field to enable setting a defaulted service name in the zipkin spans to replace the unknown-service-name existing behavior,

Fixes: #39
Fixes: #23 (I think if I am reading it correctly)

I defaulted the string to nil as to not change any behavior of the plugin by default. Could easily set default to "Kong" if that is preferred but that would change default behavior so I didn't commit it like that.

### CODE REVIEW

**Comment by jeremyjpj0916** [comment]
Hmm the spacing looks 1 off in a spot, yet when I go to actually view the file it seems fine: https://github.com/Kong/kong-plugin-zipkin/blob/4a7f5f0a1325150bd6298392680b77b080bc2024/kong/plugins/zipkin/reporter.lua

**Comment by jeremyjpj0916** [comment]
Yeah we use this internally on a fork. Thought to drop it by 😁 .

**Comment by adriancole** [comment]
looks sensible

File: kong/plugins/zipkin/reporter.lua [hunk 1]
```diff
@@ -11,8 +11,10 @@ local zipkin_reporter_mt = {
 
 local function new_zipkin_reporter(conf)
 	local http_endpoint = conf.http_endpoint
+	local default_service_name = conf.default_service_name
 	assert(type(http_endpoint) == "string", "invalid http endpoint")
 	return setmetatable({
+                default_service_name = default_service_name;
```
Comment (bungle) [inline kong/plugins/zipkin/reporter.lua]:
```suggestion
               default_service_name = default_service_name;
```
(style fix)

**Comment by bungle** [comment]
@jeremyjpj0916 could you please rebase this to get the conflict solved? I will be merging this and releasing a new version then.

**Comment by jeremyjpj0916** [comment]
Meh, closing this one for: https://github.com/Kong/kong-plugin-zipkin/pull/45 , was easier for me.

**Pull request closed without merging**